src/localization/fi.json

```diff
@@ -50,7 +50,9 @@
     "barChartDescription": "Yhteenveto aikakirjauksista aikavälin mukaan",
     "pieChartDescription": "Valitse yksittäinen aikakirjaus tai aikaväli",
     "byrange": "Aikavälin mukaan",
-    "selectTimespan": "Valitse aikaväli"
+    "viewAllTimeEntries": "Henkilöstön aikakirjaukset ja saldot",
```

Comment:
I'd add "Kaikki" at the beginning `"viewAllTimeEntries": "Kaikki henkilöstön aikakirjaukset ja saldot",`

Reply:
Updated string